Worker.php

```diff
@@ -158,23 +159,24 @@ function renameEventsLog($path) {
 }
 
 /**
- * Consumes the events file
- * - fires the rolling curl function
+ * Consumes the events file, adds the sent tstamp
+ * The stm_buffer is:
```

Comment:
`stm_buffer` is an odd name to me. It sounds like a buffer of sent timestamps, when it's actually a buffer of events with `stm` attached. Perhaps rename to `payload_with_stm_buffer` or just `final_payload_buffer` which is more generic.